apps/admin-gui/src/app/vos/pages/member-detail-page/member-resources/member-resources.component.html

```diff
@@ -0,0 +1,27 @@
+<div *ngIf="!loading">
```

Comment:
Well yes, but not like this. We still want to show the user on what page he is. When loading just hide the table. Take look at the vo-groups.component.html